### DISCUSSION TITLE
GitHub Trending is the reason that I visit GitHub every morning

### DISCUSSION THREAD

**Discussion by aliengator8** [community/community#31710] [Other Feature Feedback, Questions, & Ideas]
GitHub doing something wrong. It may cost some disadvantages for GitHub. What do you think? They can remove marketplace instead of trending. I am sure marketplace is more low usage

**Comment by pmg1991** [comment]
Please keep "Trending Repositories and Trending Developers"

**Comment by mo-vic** [comment]
I agree, I visit Treading almost everday and I do learned a lot from this page.

**Comment by skippa** [comment]
Me too! One of my favourite things to check each day is tending repos. I've found so many great projects here, please keep it!!!
